crates/modelardb_server/src/context.rs

```diff
@@ -356,11 +365,20 @@ impl Context {
     ) -> Result<SchemaRef, ModelarDbError> {
         let database_schema = self.default_database_schema()?;
 
-        let table = database_schema.table(table_name).await.ok_or_else(|| {
-            ModelarDbError::DataRetrievalError(format!(
-                "Table with name '{table_name}' does not exist."
-            ))
-        })?;
+        let table = database_schema
+            .table(table_name)
+            .await
+            .map_err(|e| {
```

Comment:
Very minor but maybe change to `|error|` to be consistent.

Reply:
Fixed.